.github/workflows/check-jsdocs-comment.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
```

Comment:
Please add Docstrings to all your functions and also at the top of your script like [countline.py](https://github.com/PalisadoesFoundation/talawa-api/blob/develop/.github/workflows/countline.py)

This script was created to ensure that our code base has correct documentation. The script should also have good internal documentation too.

Make sure the script is pydocstring compliant